app/javascript/hyacinth_ui_v1/components/digital_objects/digitalObjectInterface.js

```diff
@@ -53,15 +53,17 @@ const digitalObjectInterface = WrappedComponent => class extends React.Component
                 <Tab to={`/digital_objects/${id}/rights`} name="Rights" />
 
                 {
-                  (data.digitalObjectType === 'item') && (
+                  (data.digitalObjectType === 'item') ? (
                     <Tab to={`/digital_objects/${id}/children`} name="Manage Child Assets" />
                   )
+                    : <></>
```

Comment:
Are the empty elements important? I'm assuming the `to` attribute means we don't care about indexes of the tabs?

Reply:
The empty elements were the only way I found to get rid of a JS warning I was seeing.  When they weren't here before, the <Tabs> parent kept complaining about receiving non-element values for one or more of its children.

And yes, I'm pretty sure the tab index doesn't matter here, but I'd feel slightly better if @cgalarza confirms because she's used it more than I have.

Reply:
I think we should go back to how I originally implemented this bit. 

The warnings are going to appear because of my propTypes validation. The root issue of that warning is how I am defining the types and not how this is specifically implemented. The propTypes defined in the `Tabs` component need to be able to handle a use case like this. I use these tab components in other places so it might come up again.

Reply:
I'm finding these warnings in the js console to be distracting while I'm doing js development, and they're thrown on the **error** level so they look like errors while I'm developing.  Would it be okay if we ticket the changes you described and when they're in place, we can remove this temporary fix?  Or if you want to push a commit onto this branch, feel free, and we can remove this now.

Reply:
I would rather ticket or remove the propType check entirely.

Reply:
Sounds good!  Are you okay with creating the ticket because you best understand the changes you want us to make?

Reply:
I can create it.